src/main/kotlin/com/google/container/tools/diagnostics/GoogleFeedbackErrorReporter.kt

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.diagnostics
```

Comment:
I just realized this class is in the root project module - this is fine if it is only going to be internally used by IDE as an extension, but attempt to use it from other modules would introduce circular dependencies - does it makes sense to move it to `core` if we think we might need to use it directly?

Reply:
I don't think it would ever be used directly. The reporter is registered in plugin.xml as an `errorHandler` extension point. Afterwards, the IDE then internally invokes our implementation during any crash reporting.

I could be missing something though - did you have a specific case in mind?

Reply:
No, you're correct - if it's not going to be used other than by IDE, it's good to have in root.